edgecast/waf/common_types.go

```diff
@@ -19,6 +19,19 @@ type AddRuleResponse struct {
 	Errors []WAFError
 }
 
+// DeleteRuleResponse contains the response from the WAF API when deleting a new rule
```

Comment:
Maybe we should use the struct composite we learn yesterday.

Reply:
Agreed with @changgyu-oh , we could have a base `WAFResponse` struct with `Succcess` `Status` and `Errors`, then `AddRuleResponse` can add `ID` and `WAFResponse` as properties

Reply:
Created WAFResponse Class and updated response classes to include the base class